src/stores/client.rs

```diff
@@ -60,15 +60,17 @@ impl ClientStore for sqlx::PgPool {
         let mut notification_query_builder =
             sqlx::QueryBuilder::new("DELETE FROM public.notifications WHERE client_id = ");
         notification_query_builder.push_bind(id);
-        notification_query_builder.push("and tenant_id = ");
+        notification_query_builder.push(" and tenant_id = ");
         notification_query_builder.push_bind(tenant_id);
         let notification_query = notification_query_builder.build();
 
         self.execute(notification_query).await?;
 
+        println!("Deleting client {}", id);
+
         let mut query_builder = sqlx::QueryBuilder::new("DELETE FROM public.clients WHERE id = ");
         query_builder.push_bind(id);
-        query_builder.push("and tenant_id = ");
+        query_builder.push(" and tenant_id = ");
```

Comment:
Why spaces?

Reply:
did this work before? It failed unless there was a space for me.

Reply:
Should work, but happy to merge with this and investigate later 😄